components/forms/DonationForm.js

```diff
@@ -1,207 +1,94 @@
-                  value={amount}
-                  onClick={this.handleInputChange}
-                  className={selected ? 'selected' : ''}
-                >
-                  ${amount}
-                </button>
-              );
-            })}
-          </div>
-          <div className="donation-form__other-amount">
-            <div className="donation-form__other-amount__amount-sign">$</div>
-            <input name="amount" type="text" pattern="\d+(\.\d{2})?" onChange={this.handleInputChange} />
-          </div>
+        <div className="donation-form__field donation-form__field--medium">
+          <label htmlFor="lastName">
+            Last Name
+            <input name="lastName" type="text" onChange={handler} value={formState.lastName} />
+          </label>
         </div>
-        <div className="donation-form__row">
-          <div className="donation-form__field">
-            <label htmlFor="includeTax">
-              <input name="includeTax" type="checkbox" onClick={this.handleInputChange} /> I would like to add 2.2% plus
-              $0.30 to my donation to cover PayPal processing costs.
-            </label>
-          </div>
+        <div className="donation-form__field donation-form__field--medium">
+          <label htmlFor="email">
+            Email Address
+            <input name="email" type="email" onChange={handler} value={formState.email} />
+          </label>
         </div>
-
-        <div className="donation-form__row">
-          <PaypalButton
-            env={ENV}
-            client={client}
-            commit
-            currency="USD"
-            total={this.state.total}
-            onSuccess={onSuccess}
-            onError={onError}
-            onCancel={onCancel}
+      </div>
+      <div className="donation-form__row">
+        <div>
+          {donationAmounts.map(donationAmount => {
+            const selected = donationAmount === parseInt(formState.amount);
+            return (
+              <button
+                key={donationAmount}
+                type="button"
+                name="amount"
+                value={donationAmount}
+                onClick={handler}
+                className={selected ? 'selected' : ''}
+              >
+                ${donationAmount}
+              </button>
+            );
+          })}
+        </div>
+        <div className="donation-form__other-amount">
+          <div className="donation-form__other-amount__amount-sign">$</div>
+          <input
+            name="amount"
+            type="text"
+            pattern="\d+(\.\d{2})?"
+            onChange={handler}
+            value={donationAmounts.includes(parseInt(formState.amount)) ? '' : formState.amount}
           />
         </div>
-      </Form>
-    );
-  }
+      </div>
+      <div className="donation-form__row">
+        <div className="donation-form__field">
+          <label htmlFor="includeTax">
+            <input
+              name="includeTax"
```

Comment:
This feels like a strange name for this field since it's about PayPal processing fees, not taxes.

Reply:
Agreed, I have updated this now.